Sources/PackageDescription/Package.swift

```diff
@@ -185,6 +185,24 @@ public func ==(lhs: Package.Dependency, rhs: Package.Dependency) -> Bool {
 
 // MARK: Package Dumping
 
+struct Errors {
+    /// Storage to hold the errors.
+    private var errors = [String]()
+
+    /// Adds error to global error array which will be serialized and dumped in TOML at exit.
+    mutating func add(_ str: String) {
+        errors += ["\"\(str)\""]
```

Comment:
Actually... I guess use case in question will "crash" if you pass a string containing '"' to the version initializer.